Wrappers/Python/test/test_TranslateFunction.py

```diff
@@ -131,142 +133,34 @@ def test_TranslateFunction(self):
             func.proximal_conjugate(tmp, tau, out = out_proximal_conj1)
             func_shift.proximal_conjugate(tmp, tau, out = out_proximal_conj2)
             self.assertNumpyArrayAlmostEqual(out_proximal_conj1.as_array(), out_proximal_conj1.as_array())               
-                                
             
-if __name__ == '__main__':
-#    
-    t = TestFunction()
-    t.test_TranslateFunction()    
-
-
-#    ig = ImageGeometry(4,4)
-#    tmp = ig.allocate('random_int')
-#    b = ig.allocate('random_int')
-#    scalar = 0.4
-#            
-##    f = scalar * L2NormSquared().centered_at(b)
-##    print(f.function.function)
-#    list1 = [ L2NormSquared(), scalar * L2NormSquared(), scalar * L2NormSquared(b=b)]                            
-#        
-##        for func in list_functions:
-##            
-###            if isinstance(func, ScaledFunction):
-###                func_tmp = func.function
-###            else:
-###                func_tmp = func                
-###                                
-###            if func_tmp.b is None:
-###                tmp_data = ig.allocate()
-###            else:
-###                tmp_data = b 
-##            
-##            func_tmp = func
-##            tmp_data = ig.allocate()
-##                                        
-##            res1 = func_tmp(tmp)        
-##            res2 = func_tmp.centered_at(tmp_data)(tmp)
-##            
-##            self.assertNumpyArrayAlmostEqual(res1, res2)
+                
+    def test_TranslateFunction_MixedL21Norm(self):    
         
+        ig = ImageGeometry(4,4)
         
-
+        Grad = Gradient(ig)
         
+        b = Grad.range_geometry().allocate('random_int')
+                
+        alpha = 0.4
+        fun = alpha * TranslateFunction(MixedL21Norm(), b)
+        
+        tmp_x = Grad.range_geometry().allocate('random_int')
+        
+        res1 = fun(tmp_x)        
+        res2 = alpha * (tmp_x - b).pnorm(2).sum()        
+        self.assertAlmostEqual(res1, res2)
+        print("Check call...OK")
+        
+        # TODO
+        print("Missing convex conj, prox, prox_conj")
```

Comment:
Compute analytically convex conjugate, proximal and proximal conjugate for shifted MixedL21Norm